be/src/exec/broker_reader.cpp

```diff
@@ -112,6 +113,9 @@ Status BrokerReader::open() {
         LOG(WARNING) << ss.str();
         return Status::InternalError(ss.str());
     }
+    // TODO(cmy): The file size is no longer got from openReader() method.
+    // But leave the code here for compatibility.
+    // This will be removed later.
```

Comment:
You overwrite _file_size to 0? This should be a bug

Reply:
Fixed